docs/source/define-metadata.rst

```diff
@@ -10,9 +10,46 @@ Defining metadata (meta.yaml)
 
 
 All the metadata in the conda build recipe is specified in the
-``meta.yaml`` file, as in this simple example:
+``meta.yaml`` file. See the example below: 
 
-.. literalinclude:: ../../tutorials/meta.yaml
+.. code-block:: RST

```

Comment:
This is YAML, not RST.  I think the syntax highlighting might get messed up.

Reply:
We tested it in the local html with both YAML and RST. RST was the one that looked the best.